scripts/http-vuln-cve2014-3704.nse

```diff
@@ -216,9 +216,14 @@ local function do_sql_query(host, port, uri, user)
     }
   }
   local res = http.post(host, port, uri .. "/user/login", opt, nil, r)
-  --TODO: Check return status
 
-  return user, passwd
+  if string.match( res.body, 'mb_strlen() expects parameter 1 to be string' ) then
```

Comment:
Is this the right condition? [Metasploit module](https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/multi/http/drupal_drupageddon.rb#L177-L179) simply has a check that the request succeeded and has a body of any kind. Have you tested this with a vulnerable and a patched instance?